sdv/demo.py

```diff
@@ -322,17 +322,14 @@ def _load_tabular_dummy():
     """Load a dummy tabular demo dataframe."""
     age = np.random.randint(30, 50, 12)
     age_when_joined = age - np.random.randint(0, 10, 12)
-    faker = Faker()
-    names = [faker.name() for _ in range(12)]
```

Comment:
@csala I was actually thinking that the `Unique` constraint might make more sense with the names and addresses since they would not be expected to be primary keys. Not sure what you think